helm/policy-api-crds/values.yaml

```diff
@@ -0,0 +1,5 @@
+# Overrides Helm anotations for easy CRD adoption
+chartNameOverride: "policy-meta-operator"
+chartNamespaceOverride: "policy-system"
```

Comment:
IIUC, the idea is to remove or null these out at some point?

Reply:
Yeah, I will remove these actually.